src/analytics/regionCard.js

```diff
@@ -114,6 +114,19 @@ export class RegionCard {
     }
 
     pin() {
+        let addedTool = realityEditor.spatialCursor.addToolAtScreenCenter('spatialAnalytics');
+        const frameKey = addedTool.uuid;
+        const publicData = {
+            startTime: this.startTime,
+            endTime: this.endTime,
+            summary: this.element.outerHTML,
+        };
+        const write = () => {
+            realityEditor.network.realtime.writePublicData(addedTool.objectId, frameKey, frameKey + 'storage', 'status', publicData);
+        };
+        setTimeout(write, 1000);
```

Comment:
Why does it write twice and with a delay?

Reply:
The tool added earlier in the function had to load before it starts listening to public data changes so this is a bit paranoid about it